packages/blob/src/put.ts

```diff
@@ -105,7 +121,7 @@ export function createPutMethod<
 
     const blobApiResponse = await fetch(getApiUrl(`/${pathname}`), {
       method: 'PUT',
-      body: body as BodyInit,
+      body,
```

Comment:
nice!

Reply:
for some reason I had to re-add it myself in the multipart PR, not sure why @luismeyer